dxtbx/model/scaling_model.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import absolute_import, division
+
+from abc import ABCMeta, abstractmethod
+
+class ScalingModelBaseIface(object):
```

Comment:
does this class actually add any value? basically you just say any implementation of this will have to_dict and from_dict but not much else. In Java this interface would make sense, but in python any object that has those methods has them.

Reply:
As you hint to below, most of this is copied straight from James Parkhurst's profile model interface code. Wasn't sure if it was necessary, so happy to remove it if everything will still work fine.

Reply:
Okay, I've checked and I don't actually need to use this in the rest of my code so we can remove it.